stubs/ext-ds.stub

```diff
@@ -2,51 +2,48 @@
 
 namespace Ds;
 
-use ArrayAccess;
 use Countable;
-use IteratorAggregate;
 use JsonSerializable;
 use OutOfBoundsException;
 use OutOfRangeException;
 use Traversable;
 use UnderflowException;
 
 /**
+ * @template-covariant TKey
  * @template-covariant TValue
- * @extends Traversable<mixed, TValue>
+ * @extends Traversable<TKey, TValue>
  */
 interface Collection extends Traversable, Countable, JsonSerializable
 {
 	/**
-	 * @return Collection<TValue>
+	 * @return Collection<TKey, TValue>
 	 */
 	public function copy(): Collection;
 
 	/**
-	 * @return array<TValue>
+	 * @return array<TKey, TValue>
 	 */
 	public function toArray(): array;
 }
 
 /**
  * @template TValue
  * @implements Sequence<TValue>
```

Comment:
Shouldn't we change this to

```suggestion
  * @implements Sequence <int, TValue>
```

as changed here https://github.com/phpstan/phpstan-src/pull/126/files#diff-d0f02c2f44e13de9e215efe917400723R377

Reply:
I considered it but I don't think so. Sequence can't have keys other than int.